scripts/lint-staged

```diff
@@ -70,6 +70,7 @@ if (unstaged.length !== 0) {
   }
 }
 
+console.log('RUNNING', files)
```

Comment:
Did you commit this on purpose?

Reply:
nope, I suspect the very long fight against the various tools took it's toll when it was time to cleanup for commit